shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipeline/cases/mysql/MySQLIncrementTaskRunnable.java

```diff
@@ -17,63 +17,49 @@
 
 package org.apache.shardingsphere.integration.data.pipeline.cases.mysql;
 
-import lombok.AllArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
-import org.apache.commons.lang3.tuple.Pair;
+import org.apache.shardingsphere.integration.data.pipeline.cases.base.BaseTaskRunnable;
 import org.apache.shardingsphere.integration.data.pipeline.cases.command.ExtraSQLCommand;
-import org.apache.shardingsphere.integration.data.pipeline.util.TableCrudUtil;
+import org.apache.shardingsphere.sharding.spi.KeyGenerateAlgorithm;
 import org.springframework.jdbc.core.JdbcTemplate;
 
-import java.sql.SQLException;
-import java.time.Instant;
 import java.util.LinkedList;
 import java.util.List;
 
 @Slf4j
-@AllArgsConstructor
-public final class MySQLIncrementTaskRunnable implements Runnable {
+public final class MySQLIncrementTaskRunnable extends BaseTaskRunnable {
     
-    private final JdbcTemplate jdbcTemplate;
-    
-    private final ExtraSQLCommand extraSQLCommand;
+    public MySQLIncrementTaskRunnable(final JdbcTemplate jdbcTemplate, final ExtraSQLCommand extraSQLCommand, final KeyGenerateAlgorithm keyGenerateAlgorithm) {
+        super(jdbcTemplate, extraSQLCommand, keyGenerateAlgorithm);
+    }
     
     @Override
     public void run() {
         int executeCount = 0;
-        List<Long> newPrimaryKeys = new LinkedList<>();
-        try {
-            while (executeCount < 20 && !Thread.currentThread().isInterrupted()) {
-                newPrimaryKeys.add(insertOrderAndOrderItem());
-                if (newPrimaryKeys.size() % 2 == 0) {
-                    deleteOrderAndOrderItem(newPrimaryKeys.get(newPrimaryKeys.size() - 1));
-                } else {
-                    updateOrderAndOrderItem(newPrimaryKeys.get(newPrimaryKeys.size() - 1));
-                }
-                executeCount++;
-                log.info("Increment task runnable execute successfully.");
+        List<Object> newOrderPrimaryKeys = new LinkedList<>();
+        List<Object> newOrderItemPrimaryKeys = new LinkedList<>();
+        while (executeCount < 20 && !Thread.currentThread().isInterrupted()) {
+            newOrderPrimaryKeys.add(insertOrder());
+            newOrderItemPrimaryKeys.add(insertOrderItem());
+            if (newOrderPrimaryKeys.size() % 2 == 0) {
+                deleteOrderByPrimaryKey(newOrderPrimaryKeys.get(newOrderPrimaryKeys.size() - 1));
+                deleteOrderItemByPrimaryKey(newOrderItemPrimaryKeys.get(newOrderItemPrimaryKeys.size() - 1));
+            } else {
+                updateOrderByPrimaryKey(newOrderPrimaryKeys.get(newOrderPrimaryKeys.size() - 1));
+                updateOrderItemByPrimaryKey(newOrderItemPrimaryKeys.get(newOrderItemPrimaryKeys.size() - 1));
             }
```

Comment:
Looks it could be simplified:

1, Are `newOrderPrimaryKeys` and `newOrderItemPrimaryKeys` necessary?

2, Are `newOrderPrimaryKeys.get(newOrderPrimaryKeys.size() - 1)` and `newOrderItemPrimaryKeys.get(newOrderItemPrimaryKeys.size() - 1)` necessary?